runtime/resolvers/metrics.go

```diff
@@ -27,6 +27,7 @@ type metricsResolver struct {
 	query          *metricsview.Query
 	args           *metricsResolverArgs
 	claims         *runtime.SecurityClaims
+	meta           []map[string]any
```

Comment:
This doesn't need to be a struct field. I think it can just be computed inside the `ResolveInteractive` function right before calling `runtime.NewDriverResolverResult`?